app/segments/drill/[id]/submission/input.js

```diff
@@ -310,7 +341,14 @@ function createOutputData(drillInfo, inputValues, attemptShots, uid, did) {
 
       switch (output) {
         case "target":
-          shot.target = attemptShots[j].items.target;
+          switch (drillInfo.requirements[0].type) {
+            case "inputtedPutt":
+              shot.target = inputValues[j].distance;
```

Comment:
Instead of making an exception for inputtedPutt everywhere, is it possible to make it more generic?